sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
+              label: 'ScalarDB Core を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb-by-using-kotlin',
+              label: 'Kotlin を使用する',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Cluster を使用してトランザクションを実行してみる',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster',
+              label: 'ScalarDB Cluster を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-sql-jdbc',
+              label: 'JDBC 経由で SQL を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-sql-spring-data-jdbc',
+              label: 'Spring Data JDBC 経由で SQL を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-graphql',
+              label: 'GraphQL を使用する',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Analytics を使用して分析クエリを実行してみる',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-analytics-postgresql/getting-started',
+              label: 'ScalarDB Analytics with PostgreSQL を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-samples/scalardb-analytics-spark-sample/README',
+              label: 'ScalarDB Analytics を使用する',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: '詳細',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-using-go-for-scalardb-cluster',
+              label: 'ScalarDB Cluster に Go を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-using-python-for-scalardb-cluster',
+              label: 'ScalarDB Cluster に Python を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-analytics-postgresql/installation',
+              label: 'ScalarDB Analytics with PostgreSQL をローカルにインストールする',
+            }
+          ],
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: '開発',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'develop-overview',
+          label: '概要',
+        },
+        {
+          type: 'category',
+          label: 'トランザクションを実行する',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'data-modeling',
+              label: 'データをモデル化する',
```

Comment:
```suggestion
              label: 'データモデリング',
```
It depends on whether we use 体言止め or not, though.